spec/javascript-spec.coffee

```diff
@@ -196,65 +196,108 @@ describe "Javascript grammar", ->
-
-    describe "operators with 2 characters", ->
-      it "tokenizes += correctly", ->
-        {tokens} = grammar.tokenizeLine('test += 2')
-        expect(tokens[0]).toEqual value: 'test ', scopes: ['source.js']
-        expect(tokens[1]).toEqual value: '+=', scopes: ['source.js', 'keyword.operator.js']
-        expect(tokens[2]).toEqual value: ' ', scopes: ['source.js']
-        expect(tokens[3]).toEqual value: '2', scopes: ['source.js', 'constant.numeric.js']
-
-      it "tokenizes -= correctly", ->
-        {tokens} = grammar.tokenizeLine('test -= 2')
-        expect(tokens[0]).toEqual value: 'test ', scopes: ['source.js']
-        expect(tokens[1]).toEqual value: '-=', scopes: ['source.js', 'keyword.operator.js']
-        expect(tokens[2]).toEqual value: ' ', scopes: ['source.js']
-        expect(tokens[3]).toEqual value: '2', scopes: ['source.js', 'constant.numeric.js']
-
-      it "tokenizes *= correctly", ->
-        {tokens} = grammar.tokenizeLine('test *= 2')
-        expect(tokens[0]).toEqual value: 'test ', scopes: ['source.js']
-        expect(tokens[1]).toEqual value: '*=', scopes: ['source.js', 'keyword.operator.js']
-        expect(tokens[2]).toEqual value: ' ', scopes: ['source.js']
-        expect(tokens[3]).toEqual value: '2', scopes: ['source.js', 'constant.numeric.js']
-
-      it "tokenizes /= correctly", ->
-        {tokens} = grammar.tokenizeLine('test /= 2')
+        {tokens} = grammar.tokenizeLine(operator)
+        expect(tokens[0]).toEqual value: operator, scopes: ['source.js', 'keyword.operator.' + operator  + '.js']
+
+    describe "increment, decrement", ->
+      it "tokenizes increment", ->
+        {tokens} = grammar.tokenizeLine('i++')
+        expect(tokens[0]).toEqual value: 'i', scopes: ['source.js']
+        expect(tokens[1]).toEqual value: '++', scopes: ['source.js', 'keyword.operator.increment.js']
+
+      it "tokenizes decrement", ->
+        {tokens} = grammar.tokenizeLine('i--')
+        expect(tokens[0]).toEqual value: 'i', scopes: ['source.js']
+        expect(tokens[1]).toEqual value: '--', scopes: ['source.js', 'keyword.operator.decrement.js']
+
+    describe "conditional ternary", ->
+      it "tokenizes conditional ternary", ->
+        {tokens} = grammar.tokenizeLine('test ? expr1 : expr2')
         expect(tokens[0]).toEqual value: 'test ', scopes: ['source.js']
-        expect(tokens[1]).toEqual value: '/=', scopes: ['source.js', 'keyword.operator.js']
-        expect(tokens[2]).toEqual value: ' ', scopes: ['source.js']
-        expect(tokens[3]).toEqual value: '2', scopes: ['source.js', 'constant.numeric.js']
+        expect(tokens[1]).toEqual value: '?', scopes: ['source.js', 'keyword.operator.js']
+        expect(tokens[2]).toEqual value: ' expr1 ', scopes: ['source.js']
+        expect(tokens[3]).toEqual value: ':', scopes: ['source.js', 'keyword.operator.js']
+        expect(tokens[4]).toEqual value: ' expr2', scopes: ['source.js']
+
+    describe "logical", ->
```

Comment:
describe "logical operators"
and then it "tokenizes them"

Same goes for below